### DISCUSSION TITLE
comment between list item that does not restart list

### DISCUSSION THREAD

**Discussion by vvavrychuk** [jgm/djot#243] [General]
With Djot I can put a comment at the end of list item
```
1. item 1
   {% %}
1. item 2
```
or I can put a comment at the start of list item
```
1. item 1
1. {% %}
   item 2
```

But, I cannot put a comment between list items that does not restart list
```
1. item 1
{% %}
1. item 2
```

A practical example of such a comment between list items would be temporary commenting out some of list items
```
1. item 1
{%1. item 2%}
1. item 3
```

Of course, I can put a space before commented part to make it part of previous list item

```
1. item 1
 {%1. item 2%}
1. item 3
```

But then, it is confusing if a temporary commented list item is maybe meant to be a nested item.

I think it would be great for comment to not impact semantics...

**Comment by jgm** [comment]
I think the issue here is the overloading of comments and attribute comments, which may not be a good idea. 
See #67.

(That is, I think it would be desirable to add a line comment syntax that is handled at the block parsing level.)

**Reply by dereuromark** [reply]
Refs https://github.com/jgm/djot/pull/384
Would need to check if that solves this concrete issue.
